src/views/TypingGame.vue

```diff
@@ -26,210 +26,183 @@
-        nouns[Math.floor(Math.random() * (wordIndexMax - wordIndexMin) + wordIndexMin)] + space + verbs[Math.floor(Math.random() * (wordIndexMax - wordIndexMin) + wordIndexMin)] + space + adjectives[Math.floor(Math.random() * (wordIndexMax - wordIndexMin) + wordIndexMin)] + space + nouns[Math.floor(Math.random() * (wordIndexMax - wordIndexMin) + wordIndexMin)] + period,
-        adverbs[Math.floor(Math.random() * (wordIndexMax - wordIndexMin) + wordIndexMin)] + space + nouns[Math.floor(Math.random() * (wordIndexMax - wordIndexMin) + wordIndexMin)] + space + verbs[Math.floor(Math.random() * (wordIndexMax - wordIndexMin) + wordIndexMin)] + space + items[Math.floor(Math.random() * (wordIndexMax - wordIndexMin) + wordIndexMin)] + period,
-      ];
+    function getWord(typeIndex) {
+      return present[typeIndex][Math.floor(Math.random() * (present[typeIndex].length - 0) + 0)];
+    }
 
-      /**   "Noun, Verb, Object",
-  "Noun, Verb, Adverb",
-  "Noun, Adjective, Noun, Verb",
-  "Noun, Verb, Adjective, Noun",
-  "Adverb, Noun, Verb, Object",
-  "Noun, Verb, Prepositional Phrase",
-  "Adjective, Noun, Verb, Adverb",
-  "Noun, Modal Verb, Verb, Object",
-  "Pronoun, Verb, Adverb",
-  "Noun, Verb, Infinitive" */
-  // Article? Adjective, adjective, noun, verb adverb article 
-      const currentText = ref('');
-      const userInput = ref('');
-      const userInputConstrained = ref('');
-      const gameStarted = ref(false);
-      const gameEnded = ref(false);
-      const pasteToWin = ref(false);
-      const startTime = ref(0);
-      const endTime = ref(0);
-      let curWordIndex = 0;
-      let curCharTextColor = "black";
-      let pasteDoneOnce = false;
-      const wpm = computed(() => {
-        if (!gameEnded.value) return 0;
-        const timeInMinutes = (endTime.value - startTime.value) / 60000;
-        const wordsTyped = currentText.value.split(' ').length;
-        return Math.round(wordsTyped / timeInMinutes);
-      });
+    const currentText = ref('');
+    const userInput = ref('');
+    const userInputConstrained = ref('');
+    const gameStarted = ref(false);
+    const roundsEnded = ref(true);
+    const gameEnded = ref(false);
+    const pasteToWin = ref(false);
+    const startTime = ref(0);
+    const endTime = ref(0);
+    let curWordIndex = 0;
+    let curCharTextColor = "black";
+    let pasteDoneOnce = false;
+    const wpm = computed(() => {
+      if (!gameEnded.value && !roundsEnded) return 0;
+      const timeInMinutes = (endTime.value - startTime.value) / 60000;
+      const wordsTyped = currentText.value.split(' ').length;
+      return Math.round(wordsTyped / timeInMinutes);
+    });
 
-      function startGame() {
-        currentText.value = texts[Math.floor(Math.random() * texts.length)];
-        userInput.value = '';
-        userInputConstrained.value = '';
-        gameStarted.value = true;
-        gameEnded.value = false;
+    function startGame() {
+      if (curRound.value === 0) {
```

Comment:
Can you start the game with round being greater than 0?

Reply:
Yes, this function is used for multiple rounds. I admit that rounds functionality was an afterthought and perhaps a refactor would be beneficial. startGame() will get the current round ready to go, maybe we should rename it to startRound().